bodhi/server/models/models.py

```diff
@@ -495,6 +495,7 @@ class Build(Base):
     package_id = Column(Integer, ForeignKey('packages.id'))
     release_id = Column(Integer, ForeignKey('releases.id'))
     update_id = Column(Integer, ForeignKey('updates.id'))
+    signed = Column(Boolean, default=False)
```

Comment:
I think it makes sense to go ahead and slap a nullable=False on this guy, just to be opinionated about it. (I think a lot of fields should have this, but I can deal with that another day).